docs/docusaurus/docs/contributing/contributing_github.md

```diff
@@ -42,7 +42,7 @@ If you want to change documentation, but not code, we suggest using the GitHub m
 
 #### 4. Sign the CLA
 
-* If this is your first contribution to Great Expectations, You will see a comment from the “CLA Bot” that asks you to complete the Contributor Licence Agreement form.
+* If this is your first contribution to Great Expectations, you will see a comment from the “CLA Bot” that asks you to complete the Contributor Licence Agreement form.
```

Comment:
@dctalbot this text no longer appears in the latest version of the topic. You might want to revert this change.